### DISCUSSION TITLE
How do you "unselect" cells/column in results grid after clicking it?

### DISCUSSION THREAD

**Discussion by e3dio** [dbeaver/dbeaver#16104] [Q&A]
After you click on a cell or column I don't see how to unselect it? I want to get rid of the color highlight to the row, column, and cell

**Comment by LonwoLonwo** [comment]
Hello @e3dio 

Can you please tell me more about column "select"? What type of select do you use? 
Screens will be very helpfull!

**Reply by e3dio** [reply]
By select I mean clicking on a cell in the results grid. But now I see there is always a cell selected so you can't clear the selection

**Reply by LonwoLonwo** [reply]
Maybe you can provide a video or screenshot?

**Reply by e3dio** [reply]
There is nothing special to screenshot. The results grid has a cell selected and it is highlighted, there is no way to clear the selection

![image](https://user-images.githubusercontent.com/85405955/162267400-0c374d19-8e7c-4250-a8f5-0b9ced17b42d.png)

**Reply by LonwoLonwo** [reply]
Ctrl+click helps me in this situation.

**Reply by ShadelessFox** [reply]
To entirely clear the selection, you can switch between presentations. It's not intended, but whatever 🙂

**Reply by e3dio** [reply]
> Ctrl+click helps me in this situation

 Ctrl + click the same cell cleared the highlight and is basically what I was looking for, I am awarding you discussion answer :)
